nodeadm/internal/kubelet/config.go

```diff
@@ -43,7 +43,9 @@ func (k *kubelet) writeKubeletConfig(cfg *api.NodeConfig) error {
 	if err != nil {
 		return err
 	}
-	if semver.Compare(kubeletVersion, "v1.28.0") < 0 {
+	// tracking: https://github.com/kubernetes/enhancements/issues/3983
+	// for enabling drop-in configuration
+	if semver.Compare(kubeletVersion, "v1.31.0") < 0 {
```

Comment:
Sounds like it's aiming for beta in 1.30, so we probably don't need to wait until 1.31